users/class_tests/users_tests.py

```diff
@@ -1,78 +1,60 @@
 import pytest
-from phonenumber_field.phonenumber import PhoneNumber
-
 from users.models import User
-
-
-NAME = "user"
-EMAIL = "user@gmail.com"
-PASSWORD = "AdminU$er123"
-PHONE_NUM = "+972544651892"
-
-MISSING_REGION = "Missing or invalid default region."
-NOT_VALID_PHONE = "The phone number entered is not valid."
-NOT_INTERPRETED = "Could not interpret numbers after plus-sign."
-NOT_PHONE_NUMBER = "The string supplied did not seem to be a phone number."
-NAME_LENGTH_ERROR = "Ensure this value has at most 30 characters."
-EMAIL_ERROR = "Email is not valid"
+from phonenumber_field.phonenumber import PhoneNumber
+from .constants import NAME, EMAIL, PASSWORD, PHONE_NUM
+from .constants import MISSING_REGION, NOT_VALID_PHONE, NOT_INTERPRETED
+from .constants import NOT_PHONE_NUMBER, NAME_LENGTH_ERROR, EMAIL_ERROR
```

Comment:
instead of repeating the "from .constants" 3 times you can do it like:
```suggestion
from .constants import (
    NAME, EMAIL, PASSWORD, PHONE_NUM, MISSING_REGION, NOT_VALID_PHONE, 
    NOT_INTERPRETED, NOT_PHONE_NUMBER, NAME_LENGTH_ERROR, EMAIL_ERROR
)
```

Reply:
done